src/pages/FormPage.tsx

```diff
@@ -62,6 +65,39 @@ const FormPage = () => {
               ></tds-datetime>
             </section>
 
+            <section className="side-by-side-input">
+              <tds-text-field
+                type="text"
+                size="lg"
+                state="success"
+                label="Phone number"
+                name="phoneNumber"
+                label-position="outside"
+                no-min-width
+                placeholder="08 456 32"
+              >
+                <span slot="prefix">
+                  <tds-icon name="phone" size="16px"></tds-icon>
+                </span>
+              </tds-text-field>
+
+              <tds-text-field
+                type="text"
+                size="lg"
+                state="error"
+                label="Address"
+                name="homeAddress"
+                label-position="outside"
+                no-min-width
+                placeholder="Majorvägen 32"
+                helper="Unknown address"
+              >
+                <span slot="prefix">
+                  <tds-icon name="pin" size="16px"></tds-icon>
+                </span>
+              </tds-text-field>
```

Comment:
Should we try to set this one in an error state if the text field is empty? So in handleSubmit we check if it's empty and if so we don't submit the form and we set this one into an error state.

Reply:
Great idea, will do it ;)